lib/peep.rb

```diff
@@ -0,0 +1,25 @@
+require 'pg'
+
+class Peep
+
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
+
+    result = connection.exec("SELECT * FROM peeps;")
+    result.map { |peep| peep['message']}
+  end
+
+  def self.create(message:)
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
+
+    connection.exec("INSERT INTO peeps (message) VALUES('#{message}')")
+  end 
+end
```

Comment:
lovely neat methods, very readable and doesn't look to me like there's anything extra that shouldn't be there